posts/Ippon-USA-offers-a-helping-hand-to-those-impacted-by-natural-disasters.md

```diff
@@ -0,0 +1,32 @@
+ 
```

Comment:
1. I suggested the two tags [life-at-ippon, culture] because I thought they fit the best. Feel free to add your own.
2. The date can be blank.
3. You should find a main image for the blog, maybe something from your trip?